wicker/core/storage.py

```diff
@@ -100,7 +157,7 @@ def download_with_retries(self, bucket: str, key: str, local_path: str):
             logging.error(f"Failed to download s3 object in bucket: {bucket}, key: {key}")
             raise e
 
-    def fetch_file_s3(self, input_path: str, local_prefix: str, timeout_seconds: int = 120) -> str:
```

Comment:
I think `fetch_file_s3` is used elsewhere in other repos. to keep backward compatibility, let's keep `fetch_file_s3` and point it to call `fetch_file`.

Reply:
Done. It would be nice to add a deprecation decorator to this in the future